admin/add_project.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+require 'functions.php';
+
+$db = connectDatabase();
+$stmt = $db->query("SELECT `title`,`mini_description`,`background_image`,`project_url` FROM `projects`;");
+$projects_data = $stmt->fetch();
+
+?>
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Add project</title>
+</head>
+<body>
+
+<form method="POST" action="submit_add_project.php" enctype="multipart/form-data">
+    Title: <input type="text" name="title" value="Please enter the project title"><br/>
```

Comment:
Lines 20 to 24 consider using labels instead of free text

Reply:
changed to p's